src/blocks/content-gate-countdown/view.js

```diff
@@ -0,0 +1,39 @@
+/* globals newspack_metering_settings */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { domReady } from '../../utils';
+
+domReady( () => {
+	if ( typeof newspack_metering_settings === 'undefined' ) {
+		return;
+	}
+	const { count, gate_id } = newspack_metering_settings;
+	window.newspackRAS = window.newspackRAS || [];
+	window.newspackRAS.push( ras => {
+		const { authenticated } = ras?.getReader() || { authenticated: false };
+		if ( authenticated ) {
+			return;
+		}
+		const storeKey = 'metering-' + ( gate_id || 0 );
+		const { content } = ras?.store?.get( storeKey ) || { content: [] };
+		const countdownEl = document.querySelector( '.newspack-content-gate-countdown__countdown' );
+		if ( ! countdownEl ) {
+			return;
+		}
+		// Replace countdown for anonymous users.
+		const countdown = sprintf(
+			/* translators: 1: current number of metered views, 2: total metered views. */ __( '%1$d/%2$d', 'newspack-plugin' ),
+			content.length,
+			count
+		);
+		countdownEl.textContent = countdown;
```

Comment:
When accessing an article for the first time with anonymous metering, I'm getting 0 reads:

<img width="812" height="174" alt="Image" src="https://github.com/user-attachments/assets/31f4eebd-b314-4bcf-8f07-c0f7ec37a79e" />

We may need to listen to the [`metering_restricted` activity dispatch](https://github.com/Automattic/newspack-plugin/blob/4e1c26226035af51e41752e1188d8f58140b7ff4/src/memberships-gate/metering.js#L80) for an accurate number here